frame/staking/src/pallet/impls.rs

```diff
@@ -674,6 +693,13 @@ impl<T: Config> Pallet<T> {
 		<ErasStakersClipped<T>>::remove_prefix(era_index, None);
 		#[allow(deprecated)]
 		<ErasValidatorPrefs<T>>::remove_prefix(era_index, None);
+		#[allow(deprecated)]
```

Comment:
Do we properly benchmark this `clear_era_information` and report its weight anywhere?

Reply:
Not really. We don't benchmark anything around triggering of new era but return max weight.

https://github.com/paritytech/substrate/blob/master/frame/session/src/lib.rs#L571

Reply:
😢